AsyncDisplayKit/Private/ASDisplayNode+AsyncDisplay.mm

```diff
@@ -224,25 +225,30 @@ - (asyncdisplaykit_async_transaction_operation_block_t)_displayBlockWithAsynchro
 
       return image;
     };
-  } else if (_flags.implementsImageDisplay) {
+  } else if (_flags.implementsInstanceImageDisplay || _flags.implementsImageDisplay) {
     // Capture drawParameters from delegate on main thread
     id drawParameters = [self drawParameters];
-
+    
     displayBlock = ^id{
       __ASDisplayLayerIncrementConcurrentDisplayCount(asynchronous, rasterizing);
       if (isCancelledBlock()) {
         __ASDisplayLayerDecrementConcurrentDisplayCount(asynchronous, rasterizing);
         return nil;
       }
-
+      
       ASDN_DELAY_FOR_DISPLAY();
-
-      UIImage *result = [[self class] displayWithParameters:drawParameters isCancelled:isCancelledBlock];
+      
+      UIImage *result = nil;
+      if (_flags.implementsInstanceImageDisplay) {
```

Comment:
Should we call the pre-and post context blocks here? If not, maybe add a quick comment explaining why not?